modules/search/zlibrary.py

```diff
@@ -0,0 +1,109 @@
+"""
+OWASP Maryam!
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+from bs4 import BeautifulSoup
+
+# Using default empty strings for query, lang, ext to avoid &ext=None etc.
+meta = {
+	'name': 'Zlibrary Search',
+	'author': 'Vikas Kundu',
+	'version': '0.1',
+	'description': 'Search books and articles on Zlibrary . ',
+	'sources': ('Zlibrary',),
+	'options': (
+		('query', '', True, 'Query string to search for', '-q', 'store', str),
+		('exact', False, False, 'Search for exact query(True or False, default=False)', '-e', 'store_true', bool),
```

Comment:
Here you mention that `exact` is a `bool` but in the `examples` on line 36 you input `pdf`